mongo/src/test/java/com/redhat/lightblue/mongo/crud/MongoCRUDControllerTest.java

```diff
@@ -2851,4 +2870,11 @@ public void indexFieldsMatch_with_lightblue_path() throws Exception {
         DBObject existingIndex = new BasicDBObject("key", new BasicDBObject("a.b", 1));
         Assert.assertTrue(controller.indexFieldsMatch(index, existingIndex));
     }
+    
+    @Test
+    public void checkHealth() throws Exception{
+    	CRUDHealth healthCheck = controller.checkHealth();
+    	System.out.println("CRUDHealthCheck Details : " + healthCheck.details());
+    	Assert.assertTrue(healthCheck.isHealthy());
+    }
```

Comment:
Adding a test case where the controller is initialized with a DB Resolver with wrong port (one other than on which in memory mongo is running) which will result in refused connection and in turn an unhealthy connection. Let me know if this would be OK. can't think of any other way right now.